component/org.wso2.carbon.identity.api.otp.service.emailotp/src/gen/java/org/wso2/carbon/identity/api/otp/service/emailotp/EmailotpApi.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.com).
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.api.otp.service.emailotp;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.apache.cxf.jaxrs.ext.multipart.Attachment;
+import org.apache.cxf.jaxrs.ext.multipart.Multipart;
+import java.io.InputStream;
+import java.util.List;
+
+import org.wso2.carbon.identity.api.otp.service.emailotp.dto.Error;
+import org.wso2.carbon.identity.api.otp.service.emailotp.dto.OTPGenerateResponse;
+import org.wso2.carbon.identity.api.otp.service.emailotp.dto.OTPGenerationRequest;
+import org.wso2.carbon.identity.api.otp.service.emailotp.dto.OTPValidationRequest;
+import org.wso2.carbon.identity.api.otp.service.emailotp.dto.OTPValidationResponse;
+import org.wso2.carbon.identity.api.otp.service.emailotp.EmailotpApiService;
+
+import javax.validation.Valid;
+import javax.ws.rs.*;
```

Comment:
Let's not use * imports. Let's fix other instances as well

Reply:
The files inside the `gen` directory are `auto-generated files via `open-api` tool, I think we normally skip the styling in the `gen` directory files because they will get updated if we change the `YAML` file.